app/styles/footer.scss

```diff
@@ -1,4 +1,8 @@
+footer .container, footer ul{
+  padding: 0px;
+}
+
 footer li {
     display: inline;
-    padding: 5px;
+    padding: 0px 5px 5px 0px;
 }
```

Comment:
In the future, try to use SASS rather than just writing CSS in a SASS file. This could be:
```
footer {
  .container, ul {
    padding: 0px;
  }

  li {
    .....
  }
}
```